### PR TITLE
Ratings and Reviews - Search Functionality and Bug Fixes

### PR DESCRIPTION
**Pull request by tchitrakorn**
Summary:
- Search feature implemented 
- Bugs in the filtering feature and helpful/report feature found and addressed
- Some CSS applied

Note: 
- Bug in the form radio button still has not been addressed

### CODE REVIEW

**Comment by tchitrakorn** [comment]
Hi @matthewwrobel and @jabrake, could someone please review my PR? Thank you!

File: client/ratings_reviews/helpers.js [hunk 1]
```diff
@@ -139,6 +139,16 @@ const applyStarFilters = (reviews, starFilters) => {
   return output;
 };
 
+const applyKeyword = (reviews, keyword) => {
+  let output = reviews.filter(review => {
+    if (review.response === null) {
+      review.response = '';
+    }
+    return review.summary.includes(keyword) || review.body.includes(keyword) || review.reviewer_name.includes(keyword) || review.response.includes(keyword);
```
Comment (matthewwrobel) [inline client/ratings_reviews/helpers.js:147]: This is cool--my search only looks at the text body of the "question". Maybe I should expand it to search everything!

**Review by matthewwrobel — APPROVED** [review]

**Review by matthewwrobel — APPROVED** [review]
Looks good!

File: client/ratings_reviews/helpers.js (hunk 1 above)
Comment (tchitrakorn) [reply client/ratings_reviews/helpers.js:147]: Honestly, it wasn't quite clear what we're supposed to be searching for exactly, so I think your interpretation is fine too!

**Pull request merged**